Security/LdapUser.php

```diff
@@ -6,11 +6,12 @@
  * Time: 14:20
  */
 
-namespace Mapbender\LDAPBundle\Security\User;
+namespace Mapbender\LDAPBundle\Security;
 
 use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
 
-class LDAPUser implements UserInterface
+class LdapUser implements UserInterface, PasswordAuthenticatedUserInterface
```

Comment:
das PasswordAuthenticatedUserInterface brauchst du hier glaube ich nicht mehr, es wird ja über den Mapbender kein Passwort bereitgestellt. Siehe https://symfony.com/blog/new-in-symfony-5-3-improvements-for-security-users#decoupled-passwords-from-users

Reply:
Das scheint noch benötigt zu werden. Wenn ich es rausnehme gibt es beim Login eine Fehlermeldung.